hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/impl/ContainerSet.java

```diff
@@ -247,6 +250,16 @@ private boolean removeContainer(long containerId, boolean markMissing, boolean r
     }
   }
 
+  private void deleteContainerTable(long containerId) throws StorageContainerException {
```

Comment:
Can we rename this method. Maybe deleteFromContainerTable or removeContainerTableEntry()

Reply:
done